### ISSUE TITLE
Warning: implicit `dims=2` argument now has to be passed explicitly to specify that distances between columns should be computed

### ISSUE THREAD

**Issue by yuxiliu1995** [JuliaDynamics/RecurrenceAnalysis.jl#83]
During testing with `pkg > test RecurrenceAnalysis`, the test threw this warning:
```
┌ Warning: implicit `dims=2` argument now has to be passed explicitly to specify that distances between columns should be computed
│   caller = ip:0x0
└ @ Core :-1
```

**Comment by Datseris** [comment]
Okay, but what is the line that throws this?

**Comment by yuxiliu1995** [comment]
I don't know. The full printout is 

```
(v1.3) pkg> test RecurrenceAnalysis
   Testing RecurrenceAnalysis
 Resolving package versions...
    Status `C:\Users\DEADSC~1\AppData\Local\Temp\jl_urrXqk\Manifest.toml`
  [4fba245c] ArrayInterface v2.3.1
  [9e28174c] BinDeps v1.0.0
  [b99e7846] BinaryProvider v0.5.8
  [bbf7d656] CommonSubexpressions v0.2.0
  [34da2185] Compat v2.2.0
  [9a962f9c] DataAPI v1.1.0
  [864edb3b] DataStructures v0.17.6
  [5732040d] DelayEmbeddings v1.2.0
  [2b5f629d] DiffEqBase v6.10.0
  [01453d9d] DiffEqDiffTools v1.7.0
  [163ba53b] DiffResults v1.0.2
  [b552c78f] DiffRules v1.0.0
  [b4f34e82] Distances v0.8.2
  [ffbed154] DocStringExtensions v0.8.1
  [6e36e845] DynamicalSystemsBase v1.4.0 [`C:\Users\DeadScholar\.julia\dev\DynamicalSystemsBase`]
  [f6369f11] ForwardDiff v0.10.8
  [069b7b12] FunctionWrappers v1.0.0
  [42fd0dbc] IterativeSolvers v0.8.1
  [82899510] IteratorInterfaceExtensions v1.0.0
  [1914dd2f] MacroTools v0.5.3
  [e1d29d7a] Missings v0.4.3
  [46d2c3a1] MuladdMacro v0.2.1
  [77ba4419] NaNMath v0.3.3
  [b8a86587] NearestNeighbors v0.4.4
  [bac558e1] OrderedCollections v1.1.0
  [d96e819e] Parameters v0.12.0
  [3cdcf5f2] RecipesBase v0.7.0
  [639c3291] RecurrenceAnalysis v1.0.3 [`C:\Users\DeadScholar\.julia\dev\RecurrenceAnalysis`]
  [731186ca] RecursiveArrayTools v1.2.0
  [f2c3362d] RecursiveFactorization v0.1.0
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v0.5.2
  [f2b01f46] Roots v0.8.4
  [05bca326] SimpleDiffEq v1.0.0
  [a2af1166] SortingAlgorithms v0.3.1
  [276daf66] SpecialFunctions v0.8.0
  [90137ffa] StaticArrays v0.12.1
  [2913bbd2] StatsBase v0.32.0
  [3783bdb8] TableTraits v1.0.0
  [a2a6695c] TreeViews v0.3.0
  [30578b45] URIParser v0.4.0
  [b8865327] UnicodePlots v1.1.0
  [700de1a5] ZygoteRules v0.2.0
  [2a0f44e3] Base64  [`@stdlib/Base64`]
  [ade2ca70] Dates  [`@stdlib/Dates`]
  [8bb1440f] DelimitedFiles  [`@stdlib/DelimitedFiles`]
  [8ba89e20] Distributed  [`@stdlib/Distributed`]
  [b77e0a4c] InteractiveUtils  [`@stdlib/InteractiveUtils`]
  [76f85450] LibGit2  [`@stdlib/LibGit2`]
  [8f399da3] Libdl  [`@stdlib/Libdl`]
  [37e2e46d] LinearAlgebra  [`@stdlib/LinearAlgebra`]
  [56ddb016] Logging  [`@stdlib/Logging`]
  [d6f4376e] Markdown  [`@stdlib/Markdown`]
  [a63ad114] Mmap  [`@stdlib/Mmap`]
  [44cfe95a] Pkg  [`@stdlib/Pkg`]
  [de0858da] Printf  [`@stdlib/Printf`]
  [3fa0cd96] REPL  [`@stdlib/REPL`]
  [9a3f8284] Random  [`@stdlib/Random`]
  [ea8e919c] SHA  [`@stdlib/SHA`]
  [9e88b42a] Serialization  [`@stdlib/Serialization`]
  [1a1011a3] SharedArrays  [`@stdlib/SharedArrays`]
  [6462fe0b] Sockets  [`@stdlib/Sockets`]
  [2f01184e] SparseArrays  [`@stdlib/SparseArrays`]
  [10745b16] Statistics  [`@stdlib/Statistics`]
  [4607b0f0] SuiteSparse  [`@stdlib/SuiteSparse`]
  [8dfed614] Test  [`@stdlib/Test`]
  [cf7118a7] UUIDs  [`@stdlib/UUIDs`]
  [4ec0a83e] Unicode  [`@stdlib/Unicode`]
┌ Warning: implicit `dims=2` argument now has to be passed explicitly to specify that distances between columns should be computed
│   caller = ip:0x0
└ @ Core :-1
Test Summary: | Pass  Total
Sine wave     |   15     15
Test Summary: | Pass  Total
White noise   |   15     15
Test Summary:   | Pass  Total
Hénon (chaotic) |   15     15
Test Summary:    | Pass  Total
Hénon (periodic) |   15     15
Test Summary:         | Pass  Total
Recurrence structures |   64     64
   Testing RecurrenceAnalysis tests passed
```

**Comment by Datseris** [comment]
@heliosdrm If you know anything let me know :)

**Comment by heliosdrm** [comment]
Thanks for filing the issue.

That warning comes from Distances.jl:

https://github.com/JuliaStats/Distances.jl/blob/f69f7888c92458ae671c893d079ecf5fc8d8accd/src/generic.jl#L125-#L126

Looking at the CI logs, it started to happen here:
https://travis-ci.org/JuliaDynamics/RecurrenceAnalysis.jl/builds/527001735

This was the first time it was tested with Distances 0.8. So it's just a matter of fixing the lines where `pairwise` is called. I'll look at it asap.

**Comment by Datseris** [comment]
awesome, I knew you would find it immediatelly :D

**Comment by heliosdrm** [comment]
Closed by #84
